### PR TITLE
Bump to Lumino 2ᵅ⁵

### PR DESCRIPTION
**Pull request by afshin**
Updates Lumino 2 from `alpha.1` to `alpha.5`

### CODE REVIEW

**Comment by jtpio** [comment]
> Just restarted the UI tests check to make sure they still pass: https://github.com/jupyterlab/jupyterlab/runs/8248244462?check_suite_focus=true

Another "cancelled" after the restart, so there might be something to look into:

image

File: packages/shortcuts-extension/package.json [hunk 1]
```diff
@@ -44,13 +44,13 @@
...
-    "@lumino/commands": "^2.0.0-alpha.1",
-    "@lumino/coreutils": "^2.0.0-alpha.1",
-    "@lumino/disposable": "^2.0.0-alpha.1",
-    "@lumino/domutils": "^2.0.0-alpha.1",
+    "@lumino/algorithm": "^2.0.0-alpha.4",
+    "@lumino/commands": "^2.0.0-alpha.4",
+    "@lumino/coreutils": "^2.0.0-alpha.4",
+    "@lumino/disposable": "^2.0.0-alpha.4",
+    "@lumino/domutils": "^2.0.0-alpha.4",
     "@lumino/keyboard": "^1.8.1",
```
Comment (fcollonval) [inline packages/shortcuts-extension/package.json:52]: It seems we missed that one in the first PR:

```suggestion
    "@lumino/keyboard": "^2.0.0-alpha.4",
```

**Comment by afshin** [comment]
D'oh! Thanks @fcollonval!

**Comment by fcollonval** [comment]
@afshin I'm gonna make another release of Lumino 2-alpha that include the use of `setTimeout`. So we can test it before backporting it in Lumino 1 and JLab 3.x as @krassowski mentions it may introduce some race conditions.

**Comment by fcollonval** [comment]
The remaining integration failure is certainly related to this PR - the tab bar of the main area is displayed when switching to simple mode.

**Comment by fcollonval** [comment]
@afshin there is a bug with the tab bars update. If you run this PR, open two tabs and then close one of them, somehow the tab bar for the closed item and the add button are not removed.

I did not find the root cause. But I would bet for virtualdom because the tab title is correctly removed from the tabbar list.

**Comment by afshin** [comment]
I still haven't resolved the tab bar bug, but this is another bug that affects the current Lumino alpha, so we should not merge this PR without it:
- https://github.com/jupyterlab/lumino/pull/411

**Comment by fcollonval** [comment]
@afshin PR merged, new alpha.6 release done and this PR has been updated.

**Review by fcollonval — APPROVED** [review]
Thanks @afshin

**Pull request merged**